UnityProject/Assets/Scripts/Core/Input System/KeyboardInputManager.cs

```diff
@@ -220,7 +220,7 @@ private bool CheckComboEvent(KeyCombo keyCombo, KeyEventType keyEventType = KeyE
 		{ KeyAction.ChatLocal,		() => { ChatUI.Instance.OpenChatWindow(ChatChannel.Local); }},
 		{ KeyAction.ChatRadio,		() => { ChatUI.Instance.OpenChatWindow(ChatChannel.Common); }},
 		{ KeyAction.ChatOOC,		() => { ChatUI.Instance.OpenChatWindow(ChatChannel.OOC); }},
-		{ KeyAction.ToggleAHelp,    () => { ChatUI.Instance.OnAdminHelpButton(); }},
+		{ KeyAction.ToggleAHelp,    () => { ChatUI.Instance.OnHelpButton(); }},
```

Comment:
what window does this show?

Reply:
It's showing a window with two buttons  one for ahelp and another for mhelp, let me grab a picture

Reply:
https://crxb.cc/gxva07d top right of here is what's shown